### ISSUE TITLE
NameError: name 'urlretrieve' is not defined

### ISSUE THREAD

**Issue by fumihiko-takahashi** [MKuranowski/pyroutelib3#4]
Hi! 
I am getting the following error when trying to find a node.
I am using python 3.6 on Ubuntu14.04.
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-2-f15da053965e> in <module>()
----> 1 start = router.findNode(35.6857041,139.7321134)
      2 end = router.findNode(35.6822175,139.7376716)

~/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py in findNode(self, lat, lon)
    397         """Find the nearest node that can be the start of a route"""
    398         # Get area around location we're trying to find
--> 399         self.getArea(lat, lon)
    400         maxDist, closestNode = math.inf, None
    401 

~/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py in getArea(self, lat, lon)
    214 
    215         if downloadedSecondsAgo >= self.expire_data:
--> 216              urlretrieve("https://api.openstreetmap.org/api/0.6/map?bbox={0},{1},{2},{3}".format(_tileBoundary(x, y, 15)), filename)
    217 
    218         self.loadOsm(filename)

NameError: name 'urlretrieve' is not defined
```

I think you need import `urllib.request`, and use `urllib.request.urlretrieve`.

**Comment by MKuranowski** [comment]
Lol how could I miss this while testing?

Sorry for that, fix (with version 1.0.post1) released…

**Comment by Michael-VT** [comment]
I try

import urllib.request as tr
urllib.request.urlretrieve.

it works
